test/run_tests.pl

```diff
@@ -177,6 +189,7 @@ sub find_matching_tests {
                         print $output_buffer if !$is_ok;
                         print "\n".$self->as_string
                             if !$is_ok || $failure_verbosity == 2;
+                        print "\n--------------------------------------------------------------------------------" if !$is_ok;
```

Comment:
The basis for the perl framework is [TAP](http://testanything.org/) (Test Anything Protocol), and while not *currently* mandatory, it's encouraged to display everything that's not strictly part of the protocol as comments (i.e. lines starting with `#`).  We haven't been absolutely strict ourselves (it's been a learning process), but are getting better...  this lines goes in the other direction.

Reply:
Good point - no problem, now the separator starts with `# `.